packages/venia-ui/lib/components/SearchBar/autocomplete.js

```diff
@@ -105,6 +105,13 @@ const Autocomplete = props => {
                 },
                 { resultCount: resultCount }
             )
+        )
+        .set(
+            'CHARACTER_NOT_ENOUGH',
+            formatMessage({
+                id: 'autocomplete.characterNotEnough',
+                defaultMessage: 'Must search for at least 3 characters'
```

Comment:
First, my apologies, because I know this is the message suggested in the ticket, which is an improvement from Luma; but after reviewing this code this isn't quite accurate. Technically you **can** search by hitting enter, GraphQL is not imposing this search limit, our implementation is.

@schensley @jcalcaben - may need some help on this copy 😄 
- `Enter at least three characters to load results`
- `Search term must be at least three characters`

Reply:
Maybe something like:

> Search term must be at least 3 characters

Reply:
@jcalcaben I updated as your recommend